css/style.css

```diff
@@ -9,285 +14,826 @@ body {
-        } 
-
-        @media (max-width: 768px) { 
-            .girl {
-                background-image: none;
-            }
-        }
- 
-        @media (max-width: 768px) { 
-            .new-сollection {
-               background: url(../img/girl2.jpg) no-repeat;
-               background-size: cover contain;      
-               background-position: 0 36%;        
-           }
-        }
-
-        @media (max-width: 768px) { 
-            .title {
-                font-size: 50px;
-                line-height: 51px;
-                position: absolute;
-                left: 5%;
-                top: 60%;
-                width: 297px;
-                height: 103px;
-           }
-        }
-
-        @media (max-width: 768px) { 
-            .galka {
-                position: absolute;
-                left: 170px;
-                top: 625px;
-           }
-        }
-       
\ No newline at end of file
+/* Cтили 2й секции ниже */
+
+.bag {
+  background: #fae8e4;
+  height: 900px;
+  margin-top: 0;
+  width: 1440px;
+}
+
+.bag__wight {
+  position: absolute;
+  width: 398px;
+  height: 415px;
+  margin-left: 137px;
+  top: 1117px;
+}
+
+.bag__pink {
+  position: absolute;
+  width: 582px;
+  height: 844px;
+  margin-left: 231px;
+  top: 928px;
+}
+
+.card {
+  position: absolute;
+  width: 488px;
+  height: 556px;
+  margin-left: 879px;
+  top: 1068px;
+  background: #fff;
+}
+
+.bag__color {
+  position: absolute;
+  top: 375px;
+  left: 55px;
+}
+
+.color-wight,
+.color-blue,
+.color-green {
+  width: 30.18px;
+  height: 30px;
+  border-radius: 50%;
+  border: none;
+}
+
+.color-pink {
+  width: 38px;
+  height: 38px;
+  border-radius: 50%;
+}
+
+.color-wight {
+  background: #ded;
+}
+
+.color-pink {
+  background:#f29f9f;
+  border: 1px solid #f29f9f;
+}
+
+.color-blue {
+  background: #a8bff9;
+}
+
+.color-green {
+  background:#90bd8f;
+}
+
+span {
+  position: absolute;
+  width: 125px;
+  height: 57px;
+  left: 288px;
+  top: 365px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 56px;
+  color: #000;
+}
+
+.button-minus,
+.button-plus {
+  border: none;
+}
+
+.button-number {
+  position: absolute;
+  width: 100px;
+  height: 44px;
+  left: 55px;
+  top: 468px;
+  border: 1.5px solid #000;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.bag__button {
+  position: absolute;
+  width: 253px;
+  height: 44px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+  left: 170px;
+  top: 468px;
+  background: #fff;
+}
+
+.bag__link {
+  position: absolute;
+  width: 299px;
+  height: 58px;
+  margin-left: 884px;
+  top: 1663px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #000;
+}
+
+.bag__arrow {
+  position: absolute;
+  width: 241px;
+  margin-left: 1119.5px;
+  top: 1665px;
+}
+
+
+/* Третья секция */
+
+.total-look {
+  position: absolute;
+  width: 1440px;
+  height: 900px;
+  top: 1800px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)